.github/workflows/publish-release.yml

```diff
@@ -17,11 +17,9 @@ jobs:
       run: |
         cd S2VX.Desktop
         dotnet publish -p:PublishProfile=win-x64
-        zip -j S2VX.zip bin/Release/netcoreapp3.1/publish/S2VX.exe
-        zip -r S2VX.zip Stories
+        Compress-Archive -Path ./Stories, ./bin/Release/netcoreapp3.1/publish/S2VX.exe -DestinationPath ./S2VX.zip
```

Comment:
Is this rebased to the latest master? Why are we changing this back to Compress-Archive?

Reply:
Oh I forgot to undo the merge revert. Git won't let me merge master in again so I'll try to revert the revert